### PR TITLE
[TAS-365] ✨ Add trending NFT in index page

### PR DESCRIPTION
**Pull request by nwingt**
### Desktop
image
image

### Mobile
image

### CODE REVIEW

File: src/constant/index.js [hunk 1]
```diff
@@ -203,6 +203,9 @@ export const NFT_DISPLAY_STATE = {
   DEFAULT: 'default',
 };
 
+export const NFT_CLASS_LATEST_DISPLAY_COUNT = 10;
+export const NFT_CLASS_TRENDING_MAX_OWNER = 2;
```
Comment (williamchong) [inline src/constant/index.js:207]:
```suggestion
export const NFT_CLASS_TRENDING_MAX_PER_OWNER = 2;
```

File: src/pages/index.vue [hunk 2]
```diff
@@ -1126,11 +1126,27 @@
...
       <IndexPageHeading :text="$t('index_page_all_stories_matter_heading')" />
 
+      <div class="relative pt-[5rem] pb-[4rem]">
+        <client-only>
+          <lazy-component
+            class="absolute inset-0 -top-full pointer-events-none"
+            @show.once="fetchTrendingNFTList"
+          />
+        </client-only>
+        <IndexPageTrendingNFTList
```
Comment (williamchong) [inline src/pages/index.vue:1136]: Can we lazy load this component all together? swiper is resource heavy and below the fold

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/pages/index.vue (hunk 2 above)
Comment (nwingt) [reply src/pages/index.vue]: Updated

**Comment by nwingt** [comment]
Tested, ready for code review

**Review by AuroraHuang22 — APPROVED** [review]

**Pull request merged**